src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Debug/LaunchSettingsProvider.cs

```diff
@@ -103,8 +112,6 @@ public LaunchSettingsProvider(
     [ImportMany]
     protected OrderPrecedenceImportCollection<IDefaultLaunchProfileProvider> DefaultLaunchProfileProviders { get; set; }
 
-    protected SimpleFileWatcher? FileWatcher { get; set; }
```

Comment:
This was the only use of the `SimpleFileWatcher` class, so it should be deleted from the repo as well in this PR.

I would then also add `System.IO.FileSystemWatcher` to the list of banned APIs, with a note that we should use the CPS service instead. No other code is currently using it, but adding it to the ban list will prevent a regression here in future.